maps/nerva/nerva-3.dmm

```diff
@@ -2234,6 +2244,8 @@
 	},
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
 /obj/machinery/atmospherics/pipe/simple/hidden/supply,
+/obj/machinery/door/firedoor/border_only,
```

Comment:
These should be regular firedoors to keep them consistent, I missed that earlier. The border ones don't do anything different anyways.